### PR TITLE
Locations 15

### PR DESCRIPTION
**Pull request by amsmyth1**
This branch completes User story 15 and 16
This branch deletes unused code in the bookstores_controller
This branch refactors testing to add some edge cases

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -38,6 +38,7 @@ group :development, :test do
   # Call 'byebug' anywhere in the code to stop execution and get a debugger console
   gem 'byebug', platforms: [:mri, :mingw, :x64_mingw]
   gem 'rspec-rails'
+  gem 'rspec-pride'
```
Comment (klatour324) [inline Gemfile:41]: Yay!!! Need to get this 🙌🏻

**Review by klatour324 — COMMENTED** [review]
Looks good! I didn't notice any glaring issues here. Great work on tackling this all!
